haystack/components/preprocessors/document_splitter.py

```diff
@@ -19,6 +19,17 @@ class DocumentSplitter:
     It helps Embedders create meaningful semantic representations
     and prevents exceeding language model context limits.
 
+    The DocumentSplitter is compatible with the following DocumentStores:
+    - (Astra)[https://docs.haystack.deepset.ai/docs/astradocumentstore]
+    - (Chroma)[https://docs.haystack.deepset.ai/docs/chromadocumentstore] limited support, overlapping information is
+      not stored
+    - (ElasticSearch)[https://docs.haystack.deepset.ai/docs/elasticsearch-document-store]
+    - (OpenSearch)[https://docs.haystack.deepset.ai/docs/opensearch-document-store]
+    - (PGVector)[https://docs.haystack.deepset.ai/docs/pgvectordocumentstore]
+    - (PineCone)[https://docs.haystack.deepset.ai/docs/pinecone-document-store]
```

Comment:
I think this has the same problem of Chroma, no?

Reply:
yes, I forgot that one, thanks!